DataFormats/Headers/include/Headers/SubframeMetadata.h

```diff
@@ -1,6 +1,8 @@
 #ifndef SUBFRAMEMETADATA_H
 #define SUBFRAMEMETADATA_H
 
+#include <vector>
```

Comment:
This is currently required to avoid that including `SubframeMetadata.h` requires including `vector` before it. What we really should do is to clean up SubframeMetadata.h and remove the various `TPCTestCluster` and `ITSRawData` classes which we are currently using to mockup the timeframe.